js/libs/core.js

```diff
@@ -883,6 +888,63 @@ Fliplet.FormBuilder = (function() {
         component.methods.openFilePicker = component.methods._openFilePicker;
       }
 
+      component.methods._getFieldOptions = function() {
+        var $vm = this;
+        var fields = $vm.$parent.fields;
+
+        if ($vm.fieldOptions.length < 1) {
+          $vm.fieldOptions = fields.map(function(field) {
+            if (field._type !== 'flButtons' && field._type !== 'flAddress') {
+              return { label: field.label, disabled: false };
+            }
+          }).filter(function(field) {
+            return field;
+          });
```

Comment:
@armine-fliplet, this can be shortened to 

```suggestion
          }).filter(Boolean);
```